README.md

```diff
@@ -32,7 +32,9 @@ and assumes the environment variables that the activation script sets):
 
     source ./env/bin/activate
 
-After having created the python virtual environment and having activated it the package
+To build the project you will also need node.js. Please read the instructions for your system [here](https://nodejs.org/en/download/package-manager/).
```

Comment:
Is this accurate? I don't have node.js installed but I do have npm - they seem somewhat independent?